comps/dataprep/minio_utils.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Why we need this script?

Reply:
This change will trigger all the dataprep microservice test, so you see a list of the test in CI.

Reply:
Not required, removing it